### DISCUSSION TITLE
Import m/c defintion into stately.ai/registry/editor

### DISCUSSION THREAD

**Discussion by JayNaire** [statelyai/xstate#3232] [Q&A] [answered]
I'd love to be able to import old, hand crafted, m/c definitions into the stately editor.
I can't, there's only export functionality.
I can cut and paste into viz (truly excellent!); but then I can't visually edit the m/c.
Any suggestions please?

**Comment by davidkpiano** [comment, marked as answer]
Feel free to request this here: https://github.com/statelyai/editor-feedback/issues

**Reply by JayNaire** [reply]
Done!
Thanks David.
